netkan/netkan/notifications.py

```diff
@@ -23,7 +23,13 @@ def emit(self, record):
         self.webhook.send(f'```{fmt}```' if "\n" in fmt else fmt)
 
 
-def setup_log_handler():
+def setup_log_handler(debug=False):
+    level = logging.DEBUG if debug else logging.INFO
+    logging.basicConfig(
+        format='[%(asctime)s] [%(levelname)-8s] %(message)s', level=level
+    )
+    logging.info('Logging started for \'%s\' at log level %s', sys.argv[1], level)
```

Comment:
`sys.argv[1]` probably isn't defined for the webhooks.

Reply:
It is, it's just not useful :slightly_smiling_face: 
```bash
netkan@bef8cb5997ae:~/workspace$ gunicorn -b 0.0.0.0:5000 --access-logfile - "netkan.webhooks:create_app()"
[2019-11-12 01:44:06 +0000] [2344] [INFO] Starting gunicorn 19.9.0
[2019-11-12 01:44:06 +0000] [2344] [INFO] Listening at: http://0.0.0.0:5000 (2344)
[2019-11-12 01:44:06 +0000] [2344] [INFO] Using worker: sync
[2019-11-12 01:44:06 +0000] [2347] [INFO] Booting worker with pid: 2347
[2019-11-12 01:44:09,108] [INFO    ] Logging started for '-b' at log level 20
```